elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -192,6 +193,28 @@
   }
 }
 
+/// ===========================================================================
+/// Tab-specific SASS mixins
+/// ===========================================================================
+
+@mixin pfe-tab-panel-container($orientation: "horizontal", $border: false) {
+    margin: 0;
```

Comment:
@castastrophe set the margin to 0 to fix the issue with the panel container not aligning with the tabs on smaller displays.

Reply:
Nice!